response.go

```diff
@@ -13,7 +13,8 @@ type Response struct {
 
 // String represents a printable form of the response.
 func (r Response) String() string {
-	ret := fmt.Sprintf("Request id: %s\n", r.ID)
+	ret := fmt.Sprintf("Status: %d\n", r.Status)
+	ret+:= fmt.Sprintf("Request id: %s\n", r.ID)
```

Comment:
```
ret += fmt.Sprintf("Request id: %s\n", r.ID)
```
There's a small typo here, `:` shoud be removed from this line.

Reply:
Done. I forced pushed it away.